power-platform/admin/geo-to-geo-migrations.md

```diff
@@ -28,131 +29,146 @@ The Geo Migration feature allows customers to move their environments in a singl
-3. Fill in the details required to perform the export of the app, and then select **Export**. Once the export completes, a download should begin. The resulting file contains the app package that was selected. 
-4. Repeat these steps until all apps have been exported. 
+##### After geo-to-geo migration
+[Import the canvas apps](/power-apps/maker/canvas-apps/export-import-app#importing-a-canvas-app-package) which were exported above.
 
-After the migration: 
-1. Go to https://make.powerapps.com.
-2. Select the new environment from the environment picker in the upper-right.
-3. Select **Apps**.
-4. Select **Import canvas app**.
-5. Upload the app package file.
-6. Complete all of the import option selections, and then select **Import**.
-7. Repeat these steps until all apps have been imported. 
+#### Cloud flows
+Cloud flows can exist outside of solutions. In order to retain cloud flows, they must be exported before geo-to-geo migration then imported after geo-to-geo migration.
 
-#### For Microsoft Copilot Studio: 
+##### Before geo-to-geo migration
+[Export the cloud flows](/power-automate/export-import-flow-non-solution#export-a-flow-package) that are not in solutions.
 
--	Any Microsoft Copilot Studio chatbots must be manually exported. 
--	Some chatbots' dependent components must be manually reconfigured during or after the migration - for example, connections, environment variables, custom connectors. 
+##### After geo-to-geo migration
+[Import the cloud flows](/power-automate/export-import-flow-non-solution#import-a-flow) which were exported above.
 
-Before the migration: 
-1. Chatbots are solution-aware. You can go to https://make.powerapps.com/, navigate to the **Solutions** page, and export all chatbots'solutions - either individually or group them together in a single solution. For more information, see [Export and import bots using solutions](/power-virtual-agents/authoring-export-import-bots).
+### Dynamics 365 applications
+If you use any of the below Dynamics 365 apps, you will need to take the following actions to retain functionality after geo-to-geo migration.
 
-After the migration: 
-1. Select the new environment from https://make.powerapps.com/, and navigate to the **Solutions** page.
-2. Select **Import**, and use the file selector to pick the packages exported from the above steps.
-3. Confirm that the import was successfully completed by checking the solution contents in the target environment. 
+#### Dataverse Accelerator
+##### After geo-to-geo migration
+Uninstall the app using [PAC CLI](developer/cli/introduction.md) with the below commands:
+```
+pac solution delete --solution-name msdyn_DataverseAcceleratorApp --environment <environment URL>
+pac solution delete --solution-name DataverseAccelerator --environment <environment URL>
+pac solution delete --solution-name DataverseAccelerator_Anchor --environment <environment URL>
+```
 
-#### For Power Apps portals (must be done for each portal in the environments): 
```

Comment:
Are these steps anywhere in the updated doc? Can't seem to find them

Reply:
Power Apps portals was renamed to Power Pages. I've covered Power Pages steps in the "For components that are in solutions" section above. 

The "Reset" action called out here is the same as deleting website host. And the "Provision" action is the same as reactivating the site. I also moved both of these steps to post-migration/after migration since I found that these steps can both be done after migration rather than Reset before migration and Provision after migration. I just think it's a bit easier for customers to follow if all the steps are done together.